src/apollo/subscriptions.js

```diff
@@ -1,13 +0,0 @@
-import gql from 'graphql-tag';
-
-/** subscribe to receive new runs */
-export const NEW_RUN_SUBSCRIPTION = gql`
```

Comment:
@tynandebold - Since we are removing this subscription feature from Experiment Tracking I think it should have it's own PR. So in the future, incase we want to add it back or we want to explore this again, there's history. Especially because based on this subscription feature we were planning other features like auto-reload on Collab Experiment Tracking and so on. 

Right now because it's part of Shareable-viz -- it is not obvious.

Reply:
Fair enough.

Please review this today, if you can: https://github.com/kedro-org/kedro-viz/pull/1554